addon/-private/system/model/model.js

```diff
@@ -1115,6 +1115,14 @@ const Model = Ember.Object.extend(Ember.Evented, {
     this.notifyPropertyChange(key);
   },
 
+  notifyHasManyRemoved(key) {
```

Comment:
once https://github.com/emberjs/data/pull/5058/files#r126607364 is addressed, this goes away.